framework/base/Security.php

```diff
@@ -455,6 +459,15 @@ public function generateRandomKey($length = 32)
 
         $bytes = '';
 
+        if (function_exists('random_bytes')) {
+            try {
+                $bytes .= random_bytes($length);
```

Comment:
on success, we should return the bytes, otherwise you will run into `/dev/urandom` in both cases.